### ISSUE TITLE
fuzzer.js exception when parsing null value in example

### ISSUE THREAD

**Issue by alerodm** [apideck-libraries/portman#443]
Hey all!

I'm slowly ramping up with some pre-existent collections and contract tests, and after upgrading from portman v1.16.1 to latest (1.20.1) I started getting this error for one of our specs:
```
/Users/xxx/.nvm/versions/node/v16.18.0/lib/node_modules/@apideck/portman/dist/application/Fuzzer.js:451
            if (node.allOf || node.oneOf || node.anyOf) {
                     ^

TypeError: Cannot read properties of null (reading 'allOf')
```

I cannot share the whole spec, but after some quick debugging I believe this happens when the fuzzer tries to parse this node which comes from an example in the spec:
```
{ externalAgentId: 'agent1234', startTs: null }
```

This is the definition for `startTs`:
```
startTs:
          description: blah
          type: string
          format: date-time
          nullable: true
          example: '2021-07-08T11:20:40.237517000Z'
```

My swagger knowledge is limited, so it's unclear to me whether this is an issue with portman, or if the spec is actually not properly defined. Any help would be appreciated as I don't want to get stuck with an older version of the tool.

**Comment by thim81** [comment]
hi @alerodm 

We have a [PR](#448) in the making that would handle the "null" value better.

**Comment by alerodm** [comment]
Thank you @thim81, this is great news.

**Comment by thim81** [comment]
hi @alerodm 

🎊 We just released Portman 1.21.0, which includes the fix that solves this issue.
